### DISCUSSION TITLE
How to reference (link) to python objects [Google style] or [Numpydoc style]

### DISCUSSION THREAD

**Discussion by cmayet** [mkdocstrings/mkdocstrings#571] [Q&A] [answered]
Hi !
And big thanks to mkdocstrings developper and community !

I need help as I do not manage to reference (link) to python objects in my docstrings ! 
Using Sphinx, I would to such as documented [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects)
What is the way to obtain similar result with mkdoctrings ?
I'm using Google style or Numpydoc style.

I have a very simple example where two objects (functions in this case) lie in the same module. How to have a link to `func_a` in the documentation of `func_b` ? 

```python
def func_a():
    """What  dosctring !!

    A wonderful docstring
    """
    pass

def func_b():
    """Woooaawww docstring

    I want a reference to :py:func:`func_a`
    """
    pass
```

Any help, advice, or link to the doc welcome !
I'm a bit lost with mkodcstrings / handlers (python-legacy, python) / griffe / docstring styles documentation and I can't get it to work !

Thank you !

**Comment by pawamoy** [comment, marked as answer]
Hello! The docs are here: https://mkdocstrings.github.io/usage/#cross-references :slightly_smiling_face: 

For example:

```python
def func_a():
    """What  dosctring !!

    A wonderful docstring
    """
    pass

def func_b():
    """Woooaawww docstring

    I want a reference to [`func_a`][path.to.func_a]
    """
    pass
```

The Sphinx syntax you linked to is not supported.

**Reply by cmayet** [reply]
That's a quick answer ! Thank you !
To get it to work, i've had to add the autorefs plugin in mkdocs.yml as specified in the doc here : https://mkdocstrings.github.io/usage/#cross-references-to-any-markdown-heading

```
plugins:
- search
- autorefs
- mkdocstrings:
    [...]

```
